cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -39,7 +39,7 @@ def db_restore(temp_appliance_extended_db):
     # get DB backup file
     db_storage_hostname = conf.cfme_data.bottlenecks.hostname
     db_storage_ssh = SSHClient(hostname=db_storage_hostname, **conf.credentials.bottlenecks)
-    rand_filename = "/tmp/db.backup_{}".format(fauxfactory.gen_alphanumeric())
+    rand_filename = f"/tmp/db.backup_{fauxfactory.gen_alphanumeric()}"
     db_storage_ssh.get_file("{}/db.backup_{}".format(
         conf.cfme_data.bottlenecks.backup_path, ver), rand_filename)
     app.ssh_client.put_file(rand_filename, "/tmp/evm_db.backup")
```

Comment:
We are missing this to convert to fstring.

Reply:
There's no variable being formatted here?

Reply:
Oh I see, you mean on the line above.

Reply:
Its part of their intended design, it won't always convert.

`note: pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are anything but simple names or dotted names (as this can decrease readability).`